sys/include/phydat.h

```diff
@@ -98,6 +98,7 @@ enum {
     /* pressure */
     UNIT_BAR,       /**< Beer? */
     UNIT_PA,        /**< Pascal */
+    UNIT_HPA,       /**< hectoPascal */
```

Comment:
I prefer using another scale, instead of introducing another non-SI unit (even if it is based on a SI unit).